docs/tutorials/pydy_for_autolev_users.rst

```diff
@@ -0,0 +1,672 @@
+|                                   | One can therefore write code for  |
+|                                   | numerical computation and         |
+|                                   | visualization in the same         |
+|                                   | language.                         |
++-----------------------------------+-----------------------------------+
+||                                  ||                                  |
+| Autolev uses 1 (one) based        | Python uses 0 (zero) based        |
+| indexing. The initial element of  | indexing. The initial element of  |
+| a sequence is found using a[1].   | a sequence is found using a[0].   |
++-----------------------------------+-----------------------------------+
+||                                  ||                                  |
+| Autolev is case insensitive.      | PyDy code being Python code is    |
+|                                   | case sensitive.                   |
++-----------------------------------+-----------------------------------+
+||                                  ||                                  |
+| One can define their own commands | PyDy code is Python code, so one  |
+| in Autolev by making .R and .A    | can define functions in their     |
+| files which can be used in their  | code. This is a lot more          |
+| programs.                         | convenient.                       |
++-----------------------------------+-----------------------------------+
+||                                  ||                                  |
+| Although Autolev is still used    | The PyDy package is a suite of    |
+| for symbolic dynamics by users    | open source tools which are all   |
+| familiar with it, it is now       | brimming with vibrant             |
+| defunct.                          | communities. This is a huge plus  |
+|                                   | as the libraries are constantly   |
+|                                   | evolving to be better versions of |
+|                                   | themselves.                       |
++-----------------------------------+-----------------------------------+
+
+Rough Autolev-PyDy Equivalents
+----------------------------------
+
+The tables below give rough equivalents for some common Autolev
+expressions. **These are not exact equivalents**, but rather should be
+taken as hints to get you going in the right direction. For more detail
+read the built-in documentation on `SymPy vectors <http://docs.sympy.org/latest/modules/physics/vector/index.html>`__
+, SymPy mechanics and
+PyDy.
+
+**THIS IS AN EVOLVING DOCUMENT. If you find an error, or can fill in an
+empty box, please fix it! If there's something you'd like to see added,
+just add it.**
+
+In the tables below, it is assumed that you have executed the following
+commands in Python:
+::
+
+	from sympy.physics.mechanics import *
+	from sympy.physics.vector import *
+	from sympy import *
+	from pydy import *
```

Comment:
I'm not too fond of this, because you are left not knowing how the function is imported below. I prefer:

```python
import sympy.physics.mechanics as me
import sympy as sm
```
and explicit imports from PyDy.